<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Heydok</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vnstat GUI by Heydok</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/icon.bmp</normaloff>resources/icon.bmp</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="dateRangeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="title">
     <string>Date Range</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QDateTimeEdit" name="startDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Start date</string>
     </property>
     <property name="date">
      <date>
       <year>2021</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="endDate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>End Date</string>
     </property>
     <property name="date">
      <date>
       <year>2021</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="startDateLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Start Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="endDateLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dataBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="title">
     <string>Select Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="datatypeCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Select data type</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>Five Minute</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hourly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Daily</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monthly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yearly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top 10</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="interfaceCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Select an network Interface</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="interfaceLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Select Interface</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="datatypeLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Select Data Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="minSize">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Min total data size</string>
     </property>
     <property name="suffix">
      <string> Mb</string>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshBtn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="statusTip">
      <string>Refresh Data View</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="interfaceLbl_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Min Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tablewdgBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>551</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Data Usage</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DashDotDotLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Interface</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Download</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Upload</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Total</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="plotdataBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>226</red>
          <green>0</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>226</red>
          <green>0</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="LinkVisited">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>226</red>
          <green>0</green>
          <blue>226</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="statusTip">
     <string>Plot data in view to graph</string>
    </property>
    <property name="text">
     <string>Plot
Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="rxLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Current Download Speed in KBps</string>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="txLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Current Upload Speed in KBps</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download:</string>
    </property>
   </widget>
   <widget class="QLabel" name="uploadLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ispausedBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>452</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="statusTip">
     <string>Pause live data usage</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave_Data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/data/hd.ico</normaloff>:/data/hd.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>startDate</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>interfaceCB</tabstop>
  <tabstop>datatypeCB</tabstop>
  <tabstop>minSize</tabstop>
  <tabstop>refreshBtn</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
